package/package.json

```diff
@@ -0,0 +1,137 @@
+{
+  "name": "@commercetools/composable-commerce-test-data",
+  "version": "10.18.1",
+  "description": "Data models for commercetools platform APIs",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/test-data.git",
+    "directory": "package"
+  },
+  "keywords": [
+    "javascript",
+    "typescript",
+    "test-data"
+  ],
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/commercetools-composable-commerce-test-data.cjs.js",
+  "module": "dist/commercetools-composable-commerce-test-data.esm.js",
+  "files": [
+    "LICENSE",
+    "README.md",
+    "associate-role",
+    "attribute-group",
+    "business-unit",
+    "cart",
+    "cart-discount",
+    "category",
+    "channel",
+    "commons",
+    "core",
+    "custom-application",
+    "custom-object",
+    "custom-view",
+    "customer",
+    "customer-group",
+    "customers-search-list-my-view",
+    "discount-code",
+    "discounts-custom-view",
+    "dist",
+    "filter-values",
+    "graphql-types",
+    "inventory-entry",
+    "order",
+    "organization",
+    "organization-extension",
+    "package.json",
+    "payment",
+    "platform-limits",
+    "product",
+    "product-discount",
+    "product-projection",
+    "product-selection",
+    "product-tailoring",
+    "product-type",
+    "project",
+    "project-extension",
+    "quote",
+    "quote-request",
+    "review",
+    "shipping-method",
+    "shopping-list",
+    "staged-quote",
+    "standalone-price",
+    "state",
+    "store",
+    "tax-category",
+    "type",
+    "user",
+    "utils",
+    "zone"
+  ],
+  "preconstruct": {
+    "entrypoints": [
+      "./associate-role/index.ts",
```

Comment:
Since we're copying the source code of each model, the entrypoints should point to the `index.ts` of each model's folder.